pommerman/network/ionc/__init__.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+"""IonClient Entry point"""
+
+# Global Imports
+import ui
+from .constants import LangEN as Strings
+from .network import Network as net
+from signal import signal, SIGINT
+from gym.spaces import Discrete
+
+
+# Functions
+def init():
+    """Initialize the application"""
+
+    def exit_handler(_s, _h):
+        """Handle exiting the application"""
+        ui.fatal(ui.yellow, "Exiting..")
```

Comment:
Put this in the main (so not within the func) and call it '_exit_handler' --> So people know this it's private to this module.

Reply:
That makes more sense.